helm/g8s-cert-manager/templates/np.yaml

```diff
@@ -15,7 +15,8 @@ metadata:
 spec:
   podSelector:
     matchLabels:
-      app: {{ .Values.name }}
+      app.kubernetes.io/name: {{ template "cert-manager.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
```suggestion
      app.kubernetes.io/name: {{ template "cert-manager.name" . }}
```

Reply:
I'd suggest just going with the single selector, just because it makes it easier to understand at a glance.